QEMU 0.9.1 keeps a constant stream of data coming into the IRQ as long as
there's serial input.  (Current svn has a rate limiter, but there hasn't
been a release yet.)  This makes Linux think the device has gone nuts, and
it prints lots of spurious warnings.  Make it shut up.

diff -r 1ec969ad38de drivers/serial/8250.c
--- a/drivers/serial/8250.c	Wed Jul 02 19:29:16 2008 -0700
+++ b/drivers/serial/8250.c	Sun Jul 06 18:12:09 2008 -0500
@@ -1494,8 +1494,8 @@
 
 		if (l == i->head && pass_counter++ > PASS_LIMIT) {
 			/* If we hit this, we're dead. */
-			printk(KERN_ERR "serial8250: too much work for "
-				"irq%d\n", irq);
+//			printk(KERN_ERR "serial8250: too much work for "
+//				"irq%d\n", irq);
 			break;
 		}
 	} while (l != end);
